garnett/gsdhoomdfilereader.py

```diff
@@ -289,7 +289,7 @@ def read(self, stream, frame=None):
         :type frame: :class:`trajectory.Frame`"""
         if NATIVE:
             try:
-                gsdfile = GSDFile(stream.name, stream.mode)
+                gsdfile = gsd.fl.open(name=stream.name, mode="rb")
```

Comment:
Does this need to be `gsd.hoomd.open` as suggested by @joaander in #140 ?

Reply:
See comment above: `gsd.hoomd.open` returns a trajectory object, but garnett's current API requires that a gsdfile be passed to `GSDHoomdFrame ` objects. Not sure what is the best way to proceed.

Reply:
You can access the file object with the file attribute of the trajectory:
```
t = gsd.hoomd.open(...)
gsdfile = t.file
```

Reply:
@joaander I wasn't aware of this, will fix it soon. Thanks!